docs/OnTrack/Deployment/Google Cloud/Overview.md

```diff
@@ -14,10 +16,14 @@
 
 ## Overview
 
+The Google Cloud team's main deliverable for the project is to deploy a student-run version of
+OnTrack hosted on Google Cloud Platform (GCP) for the purpose of Thoth Tech having their own
+deployment to develop OnTrack that is separate to Deakin's deployment.
+
 ![Google Cloud - Doubtfire](https://github.com/thoth-tech/documentation/tree/main/docs/OnTrack/Deployment/Google%20Cloud/assets/GoogleCloud_Doubtfire.jpg "Google Cloud - Doubtfire")
```

Comment:
**issue:** This file is not in this PR? Could we use the relative path instead?

Reply:
Fixed in my latest commit. Thanks.

Reply:
Is this change missed on the latest commit?

Reply:
Sorry, sorted now. Thanks again!